smart-contracts/scripts/sendTransfer.js

```diff
@@ -0,0 +1,20 @@
+/*******************************************
+ *** the script just used in local test to generate a new block via trivial amount transfer
+  ******************************************/
+module.exports = async (cb) => {
+  const Web3 = require("web3");
+
+  let provider = new Web3.providers.HttpProvider(process.env.LOCAL_PROVIDER);
+
+  const web3 = new Web3(provider);
+  try {
+    const accounts = await web3.eth.getAccounts();
+    await web3.eth.sendTransaction({from: accounts[8], to: accounts[9], value: 1})
```

Comment:
Could we please loop over this 30 times in this file vs having the python file trigger this 30 times? It should make things a lot faster as truffle exec has a bunch of overhead of spinning up the compiler and checking on the artifacts and then connecting to the network.

Reply:
This loop over 30 times added into python script.

Reply:
I think we need this the other way around - there should be a yarn command that generates N blocks.  Like @ElliotFriedman said, doing this in python is really slow, but we also want to be able to do it in shell scripts.

Reply:
@juniuszhou Please add a loop inside of the javascript file to make sendTransfer.js mine 30 blocks and remove the loop in the python file. This way things will execute much faster when tests are run.

Reply:
You can use the advance block function from openzeppelin to advance blocks https://forum.openzeppelin.com/t/how-to-simulate-a-date-in-the-future-with-truffle-tests/908/3

https://docs.openzeppelin.com/test-helpers/0.5/api#advanceblock

Reply:
Can we get a method that advances N blocks, not 30?  The tests should check the balance before and after 

> You can use the advance block function from openzeppelin to advance blocks https://forum.openzeppelin.com/t/how-to-simulate-a-date-in-the-future-with-truffle-tests/908/3
> 
> https://docs.openzeppelin.com/test-helpers/0.5/api#advanceblock

Does this bignumber issue matter to us? https://github.com/OpenZeppelin/openzeppelin-test-helpers/issues/70